src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -4182,7 +4243,7 @@ resource pipeline_ToDataExplorer 'Microsoft.DataFactory/factories/pipelines@2018
                       commandTimeout: '00:20:00'
                     }
                     linkedServiceName: {
-                      referenceName: linkedService_dataExplorer.name
+                      referenceName: dataExplorerName
```

Comment:
```suggestion
                      referenceName: linkedService_dataExplorer.name
```

Reply:
Doesn't work if the resource is optional as the object will evaluate as null if we don't deploy ADX.  This causes some timing challenges too.